### PR TITLE
--silent option for more silent cronjobs

### PR DESCRIPTION
**Pull request by thanasisk**
Hi all,
the default level of logging is INFO. When berry is used in a cronjob without a special wrapper, it is a bit noisy and spamming our mailboxes. This PR introduces --silent option, which will only alert the operators when an ERROR or CRITICAL happens.

### CODE REVIEW

**Comment by CVirus** [comment]
:+1:

File: berry/cli.py [hunk 1]
```diff
@@ -181,15 +181,20 @@ def main():
                         help='Lookup AWS credentials by application ID in the given file')
     parser.add_argument('-i', '--interval', help='Interval in seconds', type=int, default=120)
     parser.add_argument('--once', help='Download credentials once and exit', action='store_true')
+    parser.add_argument('-s', '--silent', type=bool, default=False,
```
Comment (hjacobs) [inline berry/cli.py]: better use `action=store_true`, so no value is needed

File: berry/cli.py [hunk 2]
```diff
@@ -181,15 +181,20 @@ def main():
...
+                        help='silent output - only errors will be displayed')
     args = parser.parse_args()
-
-    logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
+    if args.silent:
```
Comment (hjacobs) [inline berry/cli.py]: stay DRY and do something like:

```
log_level = logging.ERROR if args.silent else logging.INFO
...

**Comment by hjacobs** [comment]
@thanasisk please fix the build: https://travis-ci.org/zalando-stups/berry/jobs/221293524

**Comment by thanasisk** [comment]
@hjacobs build is now fixed

**Comment by hjacobs** [comment]
released: https://github.com/zalando-stups/berry/releases/tag/1.0.26

**Pull request merged**